src/utils.ts

```diff
@@ -8,6 +8,8 @@ import * as core from '@actions/core'
 import * as exec from '@actions/exec'
 import * as patternHelper from '@actions/glob/lib/internal-pattern-helper'
 import {Pattern} from '@actions/glob/lib/internal-pattern'
+import {v4 as uuidv4} from 'uuid'
+import tempDirectory from 'temp-dir'
```

Comment:
🚫 **[eslint]** <[import/no-unresolved](https://github.com/import-js/eslint-plugin-import/blob/v2.25.4/docs/rules/no-unresolved.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unable to resolve path to module 'temp-dir'.